src/bp/ui-studio/src/web/views/Module/index.jsx

```diff
@@ -18,21 +18,6 @@ class ModuleView extends React.Component {
     return JSON.stringify(nextProps) !== JSON.stringify(this.props)
   }
 
-  renderLink(module) {
```

Comment:
This makes me think that we could add the possibility for modules to provide their own documentation the same way, adding docs in their assets folder

Reply:
yeah we'll need that for sure